Manual.md

```diff
@@ -781,6 +781,17 @@ with the character `r` in the `version` variable. The `distfiles`
 location will automatically be set as well as the package made to depend
 on `R`.
 
+- `gemspec` For packages that use
+[gemspec](https://guides.rubygems.org/specification-reference/) files, this is necessary
```

Comment:
I would replace semicolons with full stops in this sentence. Now for me it looks like:

For \<some condition\>, this is necessary that \<things i need to do\>.